### ISSUE TITLE
Feature Request: link and selected text to Markdown format

### ISSUE THREAD

**Issue by huyz** [reduckted/GitWebLinks#252]
Like https://github.com/kawamataryo/copy-git-link for JetBrains, it would be nice if there would be an option to generate markdown that looks like this (code snippet included if text is selected):

    [/somepackage/SomeClass.java#L42-L44](https://github.com/huyz/myrepo/blob/67b33efca188/somepackage/SomeClass.java#L42-L44)
    ```java
        public void setLast_name(String last_name) {
            this.last_name = last_name;
        }
    ```

**Comment by reduckted** [comment]
Nice idea. I'll see what I can do.

**Comment by reduckted** [comment]
Implemented for VS Code in 430cf5e60054c725689c86d1f98c6abbfbe7906c, and for Visual Studio in 87978203dbed3d64adb606995e2da70d47d40a25.

Version 2.10.0 is now available on the marketplace. 🚀

**Comment by huyz** [comment]
Awesome, thank you!
